### ISSUE TITLE
💡[Feature]: Adding project level 

### ISSUE THREAD

**Issue by vijaychandra1910** [smilewithkhushi/BasicNative#106]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

Adding level of projects .

### Use Case

where everyone can explore which type of project they want to see and which one are better for their usage.

### Priority

Medium

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by srii5477** [comment]
Hi, do you mean displaying the project's difficulty level alongside its title in the readme?  Can you assign this to me?

**Comment by srii5477** [comment]
Hi, is there any update on this?

**Comment by vijaychandra1910** [comment]
yes I am saying to show to level of the project means begginer, intermediate, advance.

**Comment by srii5477** [comment]
Ok, can you assign this to me?

**Comment by vijaychandra1910** [comment]
I will work on it.
